src/moveFile.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function copyFileContentAndMove(source, destination) {
+  try {
+    const fileContent = fs.readFileSync(source, (err) => {
```

Comment:
Use fs.readFileSync without a callback, as it is synchronous.

Reply:
Done